tests/foreman/ui/test_http_proxy.py

```diff
@@ -36,7 +36,7 @@ def function_spec_char_user(target_sat, session_auth_proxy):
 
 @pytest.mark.upgrade
 def test_positive_create_update_delete(module_org, module_location, target_sat):
-    """Create new http-proxy with attributes, update and delete it.
+    """Create new http-proxy with attributes, update, test connection and delete it.
```

Comment:
Since we're just testing connection after edit as per bug, I think this comment is little ambiguous, or do you plan to add test conenction during create as well?
And, Could you set verifies and customerscenario keywords here?

Reply:
I think the docstring says exactly what the test does: create -> update -> test connection -> delete
But I can update the keywords for sure.

Reply:
I mean, we can test connection on create and on update both, so just saying its better to be specific here that we're testing on update, unless you plan to add test connection for create as well, but its upto you now as its optional

Reply:
For test connection on create time I would rather pass separate param (like `test_connection=True) to the `create` method. We would need another Airgun PR for that.